### PR TITLE
feat(ingest): Measure event reporting latency

### PR DESCRIPTION
**Pull request by tkaemming**
This measures the difference between self-reported event timestamps and the timestamp when the server received the message.

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -851,6 +852,8 @@ def save(self, project, raw=False):
         if is_regression and not raw:
             regression_signal.send_robust(sender=Group, instance=group)
 
+        metrics.timing('event.latency', received_timestamp - recorded_timestamp)
```
Comment (tkaemming) [inline src/sentry/event_manager.py]: I'd like to tag this with project, please let me know if a.) that's viable, b.) if it is, what tag key I should be using here

**Review by tkaemming — COMMENTED** [review]
This is mostly intended to make it easier to answer the "why do my dashboard and usage graphs not match" support questions.

File: src/sentry/event_manager.py (hunk 1 above)
Comment (mattrobenolt) [reply src/sentry/event_manager.py]: @JTCunning to confirm, but `project_id` I'm pretty sure is fine.

Comment (JTCunning) [reply src/sentry/event_manager.py]: `events.latency`, `project_id` is also fine.

**Review by mattrobenolt — APPROVED** [review]

File: src/sentry/event_manager.py (hunk 1 above)
Comment (tkaemming) [reply src/sentry/event_manager.py]: Updated this to pluralize (`event` to `events`) and added the `project_id` tag with 8693eb98c3cec9e3b62ec1af45fc09bb1895e026

**Pull request merged**